ui/src/pages/frontsite/PatronProfile/PatronPastDocumentRequests/PatronPastDocumentRequests.js

```diff
@@ -62,8 +62,8 @@ export default class PatronPastDocumentRequests extends Component {
   renderEmpty = () => {
     return (
       <NoResultsMessage
-        messageHeader={'No past requests'}
-        messageContent={'You have no past literature requests.'}
+        messageHeader={'No past requests for new literatures'}
```

Comment:
small: I was not sure if literature exists in plural, dictionaries say that it is uncountable https://dictionary.cambridge.org/dictionary/english/literature

Reply:
Fixed! 👍